test/unit_tests/performance_tests.jl

```diff
@@ -8,14 +9,20 @@ import Cloudy.ParticleDistributions: integrate_SimpsonEvenFast
 
 # Adapted from CloudMicrophysics.jl
 
-function bench_press(foo, args, max_run_time; max_mem = 0.0, max_allocs = 0.0, print_args = true)
+struct CallAndReturnNothing{F}
```

Comment:
Why did you add this? Just curious what it helps with :)

Reply:
It's to avoid counting the allocations that to go the return value  -- Charlie/Dennis' suggestion :)